src/org/zaproxy/zap/extension/jython/ZapAddOn.xml

```diff
@@ -5,7 +5,7 @@
 	<description>Allows Python to be used for ZAP scripting - templates included</description>
 	<author>ZAP Dev Team</author>
 	<url></url>
-	<changes>Minor code changes</changes>
+	<changes>Add HTTP Sender template script.</changes>
```

Comment:
I believe either the removed line should be left or the version should be updated.